libbeat/kibana/testdata/extensive/fields.yml

```diff
@@ -6764,10 +6764,6 @@
             `mntr` contains the metrics reported by the four-letter `mntr`
             command.
           fields:
-            - name: hostname
-              type: keyword
-              description: >
-                ZooKeeper hostname.
```

Comment:
I assumed this is part of the metricbeat alone. 
Where are these fields used ? Or why are they part of libbeat ?

Reply:
it appears to me that `libbeat` uses `fields` configuration for a bunch of packages for testing purposes. This file in particular seems to be used for Kibana Index Pattern generation test (https://github.com/elastic/beats/blob/main/libbeat/kibana/index_pattern_generator_test.go#L122)

Reply:
Ok, we have not touched libbeat before. Although this looks fine. 
I would like the Agent team to confirm this as well. 
cc: @cmacknz